Sources/PubNubChat/Storage/CoreData/CoreDataMigrationManager.swift

```diff
@@ -63,20 +64,45 @@ class DefaultCoreDataMigrationManager: CoreDataMigrationManager {
       return
     }
     
-    for modelVersion in nextModelVersions {
+    let allVersionFiles = modelBundle.paths(
+      forResourcesOfType: "mom",
+      inDirectory: modelURL.lastPathComponent
+    )
+    
+    let sortedManagedObjectModels = allVersionFiles.compactMap() {
+      NSManagedObjectModel(contentsOf: URL(fileURLWithPath: $0))
```

Comment:
do we allow to fail here?

Reply:
Our unit tests should fail in that case. And this is what happened when I skipped the `PubNubChatModel-V2` version so it's not possible to apply a payload migration that this model version was introduced for. But what comes to my mind is improving [THIS](https://github.com/pubnub/chat-components-ios/blob/feature/payload-vsp-migration/Tests/PubNubChatTests/Storage/CoreData/CoreDataMigrationManager_Tests.swift#L169-L179) unit test. For instance, we could insert a message and check its state at the end after going through all the migrations we have so far. Thanks for raising this issue 👍